lib/ruby_lsp/disable_sorbet.rb

```diff
@@ -0,0 +1,66 @@
+# typed: true
+# frozen_string_literal: true
+
+module RubyLsp
+  # No-op all inline type assertions defined in T
+  module InlineTypeAssertions
+    def cast(value, type, checked: true)
+      value
+    end
+
+    def let(value, type, checked: true)
+      value
+    end
+
+    def must(arg)
+      arg
+    end
+
+    def absurd(value)
+      value
+    end
+
+    def bind(value, type, checked: true)
+      value
+    end
+
+    def assert_type!(value, type, checked: true)
+      value
+    end
+
```

Comment:
I think we are missing `unsafe` here (and in Shopify Types):
```suggestion

    def unsafe(value)
      value
    end
```

Reply:
Oh ok, I hadn't thought of that. Still no harm in overriding the method in case upstream implementation changes at some point, I guess.